src/search-input/src/SearchInput.js

```diff
@@ -45,6 +45,7 @@ export default class SearchInput extends PureComponent {
           paddingLeft={height}
           appearance={appearance}
           disable={disabled}
+          width={width}
```

Comment:
I think a better solution would be to add `display="block"` and `width={null}` to `TextInput` so it just stretches to fill the parent `Box`.

Reply:
I tested it, and it works if the parent Box has an explicit width.
If the parent's `Box` width is not set up, the `TextInput` wont expand to fill up space.

![image](https://user-images.githubusercontent.com/952992/36654910-6b048ab0-1a74-11e8-84de-6bca96e0d694.png)